src/store/calendars.js

```diff
@@ -2,12 +2,19 @@
  * Nextcloud - Tasks
  *
  * @author Raimund Schlüßler
+ *
  * @copyright 2018 Raimund Schlüßler <raimund.schluessler@mailbox.org>
+ *
  * @author John Molakvoæ
+ *
  * @copyright 2018 John Molakvoæ <skjnldsv@protonmail.com>
+ *
  * @author Georg Ehrke
+ *
  * @copyright 2018 Georg Ehrke <oc.list@georgehrke.com>
+ *
  * @author Thomas Citharel <tcit@tcit.fr>
+ *
  * @copyright 2018 Thomas Citharel <tcit@tcit.fr>
```

Comment:
Oh wow, that looks wrong :see_no_evil:

Reply:
Did I misunderstand something? I thought that is what was intended by https://github.com/nextcloud/eslint-config/pull/180. 😕

Reply:
Yes, but we should group the tags ahah. Not sure that is possible :disappointed: